CSharp_Course_Project/Task3/Aquatic.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharp_Course_Project.Task3
+{
+    class Aquatic : Mammal
+    {
+        public Aquatic(double age, double weight) : base(age, weight)
```

Comment:
Please put base constructor calling to the next line.

Reply:
Sorry, but I did not quite understand what you meant